docs/en/get_started.md

```diff
@@ -1,201 +1,141 @@
-|        2.1.0        |   mmcv>=0.5.9, <=0.6.1    |
-|        2.0.0        |   mmcv>=0.5.1, <=0.5.8    |
-
-**Note:** You need to run `pip uninstall mmcv` first if you have mmcv installed.
-If mmcv and mmcv-full are both installed, there will be `ModuleNotFoundError`.
-
-## Installation
-
-### A from-scratch setup script
-
-Assuming that you already have CUDA 10.1 installed, here is a full script for setting up MMDetection with conda.
-You can refer to the step-by-step installation instructions in the next section.
+# Prerequisites
+
+In this section we desmonstrate how to prepare an environment with PyTorch.
+
+MMDetection works on Linux, Windows and macOS. It requires Python 3.6+, CUDA 9.2+ and PyTorch 1.5+.
+
+```{note}
+If you are experienced with PyTorch and have already installed it, just skip this part and jump to the [next section](#installation). Otherwise, you can follow these steps for the preparation.
+```
+
+**Step 0.** Download and install Miniconda from the [official website](https://docs.conda.io/en/latest/miniconda.html).
+
+**Step 1.** Create a conda environment and activate it.
 
 ```shell
-conda create -n openmmlab python=3.7 pytorch==1.6.0 cudatoolkit=10.1 torchvision -c pytorch -y
+conda create --name openmmlab python=3.8 -y
 conda activate openmmlab
-pip install openmim
-mim install mmcv-full
-git clone https://github.com/open-mmlab/mmdetection.git
-cd mmdetection
-pip install -r requirements/build.txt
-pip install -v -e .
 ```
 
-### Prepare environment
+**Step 2.** Install PyTorch following [official instructions](https://pytorch.org/get-started/locally/), e.g.
 
-1. Create a conda virtual environment and activate it.
+On GPU platforms:
+
+```shell
+conda install pytorch torchvision -c pytorch
+```
 
-    ```shell
-    conda create -n openmmlab python=3.7 -y
-    conda activate openmmlab
-    ```
+On CPU platforms:
 
-2. Install PyTorch and torchvision following the [official instructions](https://pytorch.org/), e.g.,
+```shell
+conda install pytorch torchvision cpuonly -c pytorch
+```
 
-    ```shell
-    conda install pytorch torchvision -c pytorch
-    ```
+# Installation
 
-    Note: Make sure that your compilation CUDA version and runtime CUDA version match.
-    You can check the supported CUDA version for precompiled packages on the [PyTorch website](https://pytorch.org/).
+We recommend that users follow our best practices to install MMDetection. However, the whole process is highly customizable. See [Customize Installation](#customize-installation) section for more information.
 
-    `E.g.1` If you have CUDA 10.1 installed under `/usr/local/cuda` and would like to install
-    PyTorch 1.5, you need to install the prebuilt PyTorch with CUDA 10.1.
+## Best Practices
 
-    ```shell
-    conda install pytorch cudatoolkit=10.1 torchvision -c pytorch
-    ```
+**Step 0.** Install [MMCV](https://github.com/open-mmlab/mmcv) using [MIM](https://github.com/open-mmlab/mim).
```

Comment:
The style is different from line 160-161, line 168-169, and line 177-187. It's better to unify them